Makefile

```diff
@@ -176,7 +176,9 @@ clean_elixir:
 
 #==> Documentation tasks
 
-SOURCE_REF = $(shell tag="$(call GIT_TAG)" revision="$(call GIT_REVISION)"; echo "$${tag:-$$revision}")
+SOURCE_REF = $(shell tag="$(call GIT_TAG)"; revision="$(call GIT_REVISION)"; \
```

Comment:
Why not change `GIT_TAG` itself?

I think we can pipe the result of that to `grep` before we get the tail? We could also change which part of the tail we get.

Reply:
Good call, done!